test/views/ViewNotificationsViewSpec.scala

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views
+
+import models.viewmodels.notificationspage.NotificationsPageSingleElement
+import play.api.test.FakeRequest
+import play.twirl.api.Html
+import testdata.CommonTestData
+import testdata.CommonTestData.exampleNotificationPageSingleElement
+import views.html.view_notifications
+
+class ViewNotificationsViewSpec extends ViewSpec {
+
+  private def page(
+    submissionUcr: String = "",
+    submissionElement: NotificationsPageSingleElement,
+    elementsToDisplay: Seq[NotificationsPageSingleElement] = Seq.empty
+  ): Html =
+    new view_notifications(main_template)(submissionUcr, submissionElement, elementsToDisplay)(FakeRequest(), messages)
+
+  "Notification page" should {
+
+    "contain title" in {
+      val title =
+        page(submissionUcr = "TEST UCR", NotificationsPageSingleElement("title", "timestamp", Html("content")))
+          .getElementById("title")
+
+      val expectedTitle = messages("notifications.title", "TEST UCR")
+      title must containText(expectedTitle)
```

Comment:
For future reference you can do `title must containMessage("notification.title", "TEST UCR")`